docs/docs/integration/embedding.md

```diff
@@ -246,12 +246,12 @@ public class DashboardController {
 
 The API accepts the following parameters:
 
-| Parameter | Description                                                                                                                                                                           | Required                        |
-| --- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------|
-| resource | The name of the dashboard to embed                                                                                                                                                    | Yes                             |
-| user_email | The email of the user to embed the dashboard for                                                                                                                                      | Yes (either this or `attributes`) |
-| attributes | Json payload to be put in the access token, used to pass custom attributes to the dashboard for enforcing policies | No (either this or `user_email`)  |  
-| ttl_seconds | The time to live for the iframe URL                                                                                                                                                   | No (Default: 86400)             |
+| Parameter | Description                                                                                                                                                                                    | Required                        |
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------|
+| resource | The name of the dashboard to embed                                                                                                                                                             | Yes                             |
+| user_email | The email of the user to embed the dashboard for                                                                                                                                               | Yes (either this or `attributes`) |
+| attributes | Json payload to be put in the access token, used to pass attributes to the dashboard for enforcing policies. When using this make sure to pass all the attributes used in your security policy | No (either this or `user_email`)  |
```

Comment:
I guess `user_email` should not be required also, and the `(either this or ...` comment no longer applies? Since they're not needed for non-security policy projects

Reply:
yes made user_email optional. but the comment would be needed to indicate that only one can be set, right ?